src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submission_details.cljs

```diff
@@ -126,9 +126,43 @@
                    (:workflows props))}])))})
 
 
+(react/defc AbortButton
+  {:kill-submission
```

Comment:
At this point there's "abort", "kill", and "stop" all floating around.  Let's consolidate on one term.  Rawls uses "abort" for the endpoint, so I think we should go with that.